### DISCUSSION TITLE
关于申请音乐 api 的问题

### DISCUSSION THREAD

**Discussion by Chaochao2020** [imsyy/home#112] [General]
您好, 在部署您的项目时, 我想自己更换一下音乐api进而更换自己喜欢的音乐, 在您给出的 [GitHub](https://github.com/xizeyoupan/Meting-API#deno-deploy ) 链接中, 我查看文档说明后仍然未申请成功, 我想您方便的话能具体告诉一下如何申请这个音乐 api 嘛? 无论您回复与否, 我都不胜感激!

**Comment by Chaochao2020** [comment]
问题补充: 
1. 有自己的域名, 没有云服务器, 因为我在本地部署后会生成静态页面上传 GitHub pages, 用不到服务器的
2. 关于 GitHub 中的 Readme 中, 我不清楚 "手动部署" ,"部署到vercel"等不同方式所带来的不同.

**Comment by imsyy** [comment]
这个其实不难搞，具体就是
- fork 这个项目 https://github.com/xizeyoupan/Meting-API
- 然后前往 action 标签页开启 action
- 前往 https://dash.deno.com/ 注册并绑定你的 GitHub 账号
- 点击 [New Project](https://dash.deno.com/new) 创建一个项目，复制项目名
  ![image](https://user-images.githubusercontent.com/42232682/227828498-2b1402db-b67a-42d7-b44d-d972fa7bcb81.png)
- 在项目设置中加一个 Environment Variable，名称是 OVERSEAS，值为1
  ![image](https://user-images.githubusercontent.com/42232682/227829932-39faf9b6-d294-48d3-a75b-e024df04bf8c.png)
- 然后前往 fork 的仓库更改这个文件 [publish.yml](https://github.com/imsyy/Meting-API/blob/6e5e005fe1b3eccc0505d553e52f6b953b6d45a4/.github/workflows/publish.yml#L96)
  ![image](https://user-images.githubusercontent.com/42232682/227828258-3e1feed6-88b4-40b5-be7d-f5d3d10693af.png)
- 然后再次前往 action 标签页，此时应该已经开始进行部署了，如果没有，就再随意的改动一个文件触发部署就行了
- deno 那边应该就可以预览了

**Reply by Chaochao2020** [reply]
非常感谢!

**Comment by Kirara-Tianqing** [comment]
请问publish.yml中修改的项目名是指deno创建的项目名还是github的分支项目名？

**Comment by Kirara-Tianqing** [comment]
部署时报错![image](https://github.com/imsyy/home/assets/81161427/c109bc00-0ae2-40f2-a485-023349ebf975)
